unit/deserialize.go

```diff
@@ -38,10 +38,8 @@ const (
 	SYSTEMD_NEWLINE = "\r\n"
 )
 
-var (
-	// ErrLineTooLong gets returned when a line is too long for systemd to handle.
-	ErrLineTooLong = fmt.Errorf("line too long (max %d bytes)", SYSTEMD_LINE_MAX)
-)
+// ErrLineTooLong gets returned when a line is too long for systemd to handle.
+var ErrLineTooLong = fmt.Errorf("line too long (max %d bytes)", SYSTEMD_LINE_MAX)
```

Comment:
I don't mind that change but it seems unrelated to this commit.

Reply:
My bad, it's gofumpt. Removed from this patch.